pom.xml

```diff
@@ -204,5 +204,6 @@
   </dependencies>
   <properties>
     <java.version>1.8</java.version>
```

Comment:
Is this enough? How about the other three packages that the scanner reported? I think we should update those too to the latest version.

Reply:
```
log4j-api                              2.17.1                                     java-archive  
log4j-over-slf4j                       1.7.30                                     java-archive  
log4j-to-slf4j                         2.17.1                                     java-archive  
slf4j-log4j12                          1.7.30                                     java-archive  

```

Reply:
Here's a info about from slf4j - https://www.slf4j.org/log4shell.html